### PR TITLE
fix(forms/multi-select-tag): Improve accessibility

### PR DESCRIPTION
**Pull request by talend-sromain**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

We can't navigate in dropdown.

**What is the new behavior?**

Keyboard navigation is now working

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://390.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://390.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://390.talend.surge.sh)

File: packages/forms/src/widgets/MultiSelectTagWidget/MultiSelectTagWidget.js [hunk 1]
```diff
@@ -119,18 +126,59 @@ class MultiSelectTagWidget extends React.Component {
...
+	onComponentMount(component) {
+		this.component = component;
+	}
+
 	onKeyDown(event) {
-		if (event.which === 13) {
-			const optionsToShow = this.getOptionsToShow();
+		const optionsToShow = this.getOptionsToShow();
+		switch (event.which) {
+		case KEY_BACK_SPACE: {
```
Comment (jare-talend) [inline packages/forms/src/widgets/MultiSelectTagWidget/MultiSelectTagWidget.js]: plz use https://www.npmjs.com/package/keycode

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://390.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://390.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://390.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://390.talend.surge.sh)

File: packages/forms/src/widgets/MultiSelectTagWidget/MultiSelectTagWidget.js [hunk 2]
```diff
@@ -166,6 +211,31 @@ class MultiSelectTagWidget extends React.Component {
 		);
 	}
 
+	scrollDropDownIfRequired(selectedIndex, direction) {
+		const dropDown = this.component.querySelector('.dropdown-menu');
+		const scrollPosition = dropDown.scrollTop;
+		const dropDownHeight = dropDown.clientHeight;
+		const itemStart = (selectedIndex * DROP_DOWN_ITEM_HEIGHT) + 10;
```
Comment (jlame) [inline packages/forms/src/widgets/MultiSelectTagWidget/MultiSelectTagWidget.js]: use a variable for the 10

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://390.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://390.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://390.talend.surge.sh)

**Pull request merged**